app/controllers/renewals_controller.rb

```diff
@@ -63,6 +70,12 @@ def bulk_renewal_flash(response, type)
       )
     end
 
+    def bulk_renewal_summary_flash(response, type)
+      return unless response[type].any?
+
+      flash[type] = I18n.t("renew_all_items_summary.#{type}_html", count: response[type].length)
+    end
+
```

Comment:
This is nice! I was just doing something similar in my holds branch. There might be a way to elevate this to ApplicationController and merge our two methods. But for now this is nice and 25 is a nice number to keep cookie overflows at bay. 👍

Reply:
yes, I agree with merging or refactoring those two, let's chat about it 👍